empiric-package/empiric/core/entry.py

```diff
@@ -96,3 +96,6 @@ def flatten_entries(entries: List[Entry]) -> List[int]:
         expanded = [entry.to_tuple() for entry in entries]
         flattened = [x for entry in expanded for x in entry]
         return [len(entries)] + flattened
+
+    def __repr__(self):
+        return f'Entry("{felt_to_str(self.pair_id)}", {self.value}, {self.timestamp}, "{felt_to_str(self.source)}", "{felt_to_str(self.publisher)}")'
```

Comment:
Nice, we used to have this - must have gotten lost when going from Entry as named tuple to Class. I found it helpful to use kwarg framing i.e. `f'Entry("pair_id{felt_to_str(self.pair_id)}", value={self.value}, timestamp={self.timestamp}, source="{felt_to_str(self.source)}", publisher="{felt_to_str(self.publisher)}")'`